src/stories/Spacing.stories.js

```diff
@@ -0,0 +1,37 @@
+import { html } from 'lit-html';
+
+
+export const Spacing = () => html`
+<div>
+<!-- Your content using spacing utility classes -->
+<div class="kd-spacing__padding-top-8">kd-spacing__padding-top-8</div>
```

Comment:
Maybe just show a single example with Controls to change property, direction, and value.

Reply:
Refactored and resolved